### DISCUSSION TITLE
求一个之前的包括7.8.5版本,windows10平台

### DISCUSSION THREAD

**Discussion by wniter** [getlantern/lantern-forum-cn#693] [Q&A]
求一个之前的包括7.8.5版本，发我邮箱595436346@qq.com

**Comment by ycjdixon** [comment]
7.8.5已经发你邮箱了，不过你不用抱很大希望。现在基本上用不了了。我一直在用其他免费的VPN。@wniter

**Reply by kidokogills** [reply]
老哥，也能发我一个吗？感谢，邮箱 563415421@qq.com

**Reply by ycjdixon** [reply]
@kidokogills
已经发你了

**Reply by kidokogills** [reply]
收到，感谢
